src/aegis/src/aegis.erl

```diff
@@ -33,50 +31,47 @@ init_db(#{} = Db, Options) ->
         is_encrypted => aegis_server:init_db(Db, Options)
     }.
 
-
 open_db(#{} = Db) ->
     Db#{
         is_encrypted => aegis_server:open_db(Db)
     }.
 
-
 get_db_info(#{is_encrypted := IsEncrypted} = Db) ->
-    KeyManagerInfo = case erlang:function_exported(?AEGIS_KEY_MANAGER, get_db_info, 1) of
-        true ->
-            ?AEGIS_KEY_MANAGER:get_db_info(Db);
-        false ->
-            []
-    end,
+    KeyManagerInfo =
+        case erlang:function_exported(?AEGIS_KEY_MANAGER, get_db_info, 1) of
+            true ->
+                ?AEGIS_KEY_MANAGER:get_db_info(Db);
+            false ->
+                []
+        end,
     [{enabled, IsEncrypted}, {key_manager, {KeyManagerInfo}}].
 
-
 encrypt(#{} = _Db, _Key, <<>>) ->
     <<>>;
-
 encrypt(#{is_encrypted := false}, _Key, Value) when is_binary(Value) ->
     Value;
-
-encrypt(#{is_encrypted := true} = Db, Key, Value)
-        when is_binary(Key), is_binary(Value) ->
+encrypt(#{is_encrypted := true} = Db, Key, Value) when
+    is_binary(Key), is_binary(Value)
+->
     aegis_server:encrypt(Db, Key, Value).
 
-
 decrypt(#{} = Db, Rows) when is_list(Rows) ->
-    lists:map(fun({Key, Value}) ->
-        {Key, decrypt(Db, Key, Value)}
-    end, Rows).
+    lists:map(
+        fun({Key, Value}) ->
+            {Key, decrypt(Db, Key, Value)}
+        end,
+        Rows
+    ).
 
 decrypt(#{} = _Db, _Key, <<>>) ->
     <<>>;
-
 decrypt(#{is_encrypted := false}, _Key, Value) when is_binary(Value) ->
     Value;
-
-decrypt(#{is_encrypted := true} = Db, Key, Value)
-        when is_binary(Key), is_binary(Value) ->
+decrypt(#{is_encrypted := true} = Db, Key, Value) when
+    is_binary(Key), is_binary(Value)
+->
```

Comment:
Is there a way to not have the function arrow on a newline like this? Definitely not a fan hehe.

Reply:
That does look odd